packages/jest-changed-files/src/git.ts

```diff
@@ -29,8 +29,10 @@ const adapter: SCMAdapter = {
     const changedSince: string | undefined =
       options && (options.withAncestor ? 'HEAD^' : options.changedSince);
 
-    const includePaths: Array<Config.Path> =
-      (options && options.includePaths) || [];
+    const includePaths: Array<Config.Path> = (
+      (options && options.includePaths) ||
+      []
+    ).map(absoluteRoot => path.normalize(path.relative(cwd, absoluteRoot)));
```

Comment:
what's this change for?

Reply:
jest invoking command from **repo root of git**. e.g. `C:\my-repo`. with command
e.g. `git ls-files --other --modified --exclude-standard C:\my-repo\my-ui\`. Here is the problem.

Cygwin does not understand `C:\my-repo\my-ui`. He only knows `/cygdrive/c/my-repo/my-ui/` **or** `./my-ui/`. So if we give relative path, it will be work on linux, windows, cygwin, msys2, _whatever enviroment for people that wants linux experience on windows_

Marked lines of code, changing from `C:\my-repo\my-ui\` to `my-ui`.
Because `path.relative` can return empty string ( `path.relative("C:\\my-repo", "C:\\my-repo") // returns ''` ), normalize is used ( `path.normalize('') // returns '.'` )

Reply:
that makes sense, thanks!